### ISSUE TITLE
Feature request: enable creation of Wikidata item for advanced users

### ISSUE THREAD

**Issue by egils-consulting** [commons-app/apps-android-commons#3935]
I love finding and documenting all the treasure campsites of the world.
To do this more effectively I need an app to both create the wikidata item.and to enable uploading various types of photos to it e.g. P18 and preferably: P7418, P8517, P5252, P3451, P4717,

**Comment by misaochan** [comment]
Hi @egils-consulting , thanks for your suggestion! I agree that this feature could be useful, but a couple of concerns that I have about it:
- Creating Wikidata items is not really related to the core functionality of the app, which is based on making it easier to upload and browse images
- Vandalism could be a concern, especially as many of our app users might not be familiar with Wikidata

What do you think about these issues? 

Input from anyone else is greatly appreciated, too. :)

**Comment by egils-consulting** [comment]
Yeah, I found another venue that might be better suited, a fork of www.fruktkartan.se.
